TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -48,16 +48,27 @@ class AudioRecordingsTableViewController: UITableViewController {
     
     override func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
         let deleteAction = UIContextualAction(style: .destructive, title: "Delete") { (action, view, completionHandler) in
-            AudioManager.sharedInstance.removeFile(at: indexPath.row)
-            tableView.deleteRows(at: [indexPath], with: .automatic)
+            
+            let deleteAlertController = UIAlertController(title: "Are you sure you want to delete?", message: "You won't be able to recover this file", preferredStyle: .alert)
+            let deleteAlertAction = UIAlertAction(title: "Delete", style: .destructive, handler:  { _ in
+                AudioManager.sharedInstance.removeFile(at: indexPath.row)
+                tableView.deleteRows(at: [indexPath], with: .automatic)
+            })
+            let cancelDeleteAction = UIAlertAction(title: "Cancel", style: .cancel, handler: { _ in
+                completionHandler(false)
```

Comment:
You can leave the completion block empty here or mark it as nil

Reply:
Would I still be able to set the completionHandler as false? I needed this to get rid of the UIConextualAction UI after selecting cancel.